test/metrics/processResourcesTest.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+describe('processRequests', () => {
+	const register = require('../../index').register;
+	const processResources = require('../../lib/metrics/processResources');
+
+	beforeAll(() => {
+		register.clear();
+	});
+
+	afterEach(() => {
+		register.clear();
+	});
+
+	it('should add metric to the registry', async () => {
+		if (typeof process.getActiveResourcesInfo !== 'function') {
+			it.skip('process.getActiveResourcesInfo() is not a function');
```

Comment:
Oops, CI failed with:

```
Missing second argument. It must be a callback function. Perhaps you want to use `test.todo` for a test placeholder.

      15 | 	it('should add metric to the registry', async () => {
      16 | 		if (typeof process.getActiveResourcesInfo !== 'function') {
    > 17 | 			it.skip('process.getActiveResourcesInfo() is not a function');
         | 			   ^
      18 | 			return;
      19 | 		}
      20 | 
```

Can probably just delete this line.

Reply:
@zbjornson 🤦 thanks, deleted that. Hopefully we'll get a green CI this time!